scripts/generate_parser_test_files.py

```diff
@@ -18,7 +18,7 @@
 
 def get_test_article(enum: PublisherEnum, url: Optional[str] = None) -> Optional[Article]:
     if url is not None:
-        source = WebSource([url], publisher=enum.publisher_name)
+        source = WebSource([url], publisher=enum.publisher_name, query_parameters=enum.query_parameter)
```

Comment:
Since this is about crawling a specified URL we should not pass the query parameter here.